LEGO1/lego/legoomni/src/act1/act1state.cpp

```diff
@@ -1,34 +1,295 @@
 #include "act1state.h"
 
+DECOMP_SIZE_ASSERT(Act1State, 0x26c)
+DECOMP_SIZE_ASSERT(Act1State::NamedPlane, 0x4c)
+DECOMP_SIZE_ASSERT(Act1State::NamedTexture, 0x14)
+
+#define isle_asize(ARR) (sizeof(ARR) / sizeof((ARR)[0]))
```

Comment:
This macro would probably be useful in other locations (and should be upper cased)

Reply:
there is one here: ttps://github.com/isledecomp/isle/blob/master/util/decomp.h#L17 We can change the name of it

Reply:
Thanks, I'll re-use `_countof`. But added some parentheses around it.